guide/src/SUMMARY.md

```diff
@@ -53,7 +53,8 @@
     - [Exported Rust Types](./reference/types/exported-rust-types.md)
     - [`JsValue`](./reference/types/jsvalue.md)
     - [`Box<[T]>` and `Vec<T>`](./reference/types/boxed-slices.md)
-    - [`*const T` and `*mut T`](./reference/types/pointers.md)
+    - [`*const T`, `*mut T`](./reference/types/pointers.md)
```

Comment:
Why the change from 'and' to ','?

Reply:
Thank you for the very careful review!

I previously had this and `NonNull` in the same file, so it was "`*const T`, `*mut T` and `NonNull`", I then split it into separate files and remove "and `NonNull`".